client/src/Components/CarList.js

```diff
@@ -1,31 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import { useQuery } from "@apollo/react-hooks";
-
+import Select from 'react-select';
 import { getUserDataQuery } from "../queries/queries";
 import LoadingSpinner from "./LoadingSpinner";
 
 function displayCars(data) {
-  return data.cars.map(car => {
-    return (<li key={car.id}>
-      {car.displayName}({car.make} - {car.model})
-    </li>);
+   return data.cars.map(car => {
+      return { value: `${car.id}`, label: `${car.displayName} ${car.make} (${car.model})` }
   });
 }
 
 export default function() {
+  const [selectedOption , setSelectedOption] = useState(null)
+  const handleChange = selectedOption => {
+    setSelectedOption(selectedOption)
+    console.log(`Option selected:`, selectedOption.value);
+  }
+
   const { loading, error, data } = useQuery(getUserDataQuery);
   if (loading) {
     return <LoadingSpinner />;
   } else if (error) {
     debugger;
     return <p>Error! {error}</p>;
   } else {
-    debugger;
 
     return (
       <div>
-        <ul>{displayCars(data.userData)}</ul>
+        {console.log(displayCars(data.userData))}
```

Comment:
I suggest using `debugger;` statements for debugging purposes. This can be deleted.